crates/graphql/src/schema/scalars/mod.rs

```diff
@@ -10,6 +10,9 @@ pub(self) mod prelude {
 pub mod markers {
     pub struct StoreConfig;
     pub struct TokenMint;
+    /// Used to denote a public key field to graphql while not
+    /// indicating a relationship to any other specific object
+    pub struct Unspecified;
```

Comment:
@ray-kast I can delete this if its dumb / not useful

Reply:
I was thinking that `Wallet` sort of implies a "user" since it is linked with a twitter account, when that isn't necessarily the case here, but I deleted this `Unspecified` marker and replaced with `Wallet`